manim/scene/vector_space_scene.py

```diff
@@ -522,9 +522,6 @@ def __init__(
         include_foreground_plane=True,
         background_plane_kwargs={
             "color": GREY,
-            "axis_config": {
-                "stroke_color": LIGHT_GREY,
-            },
             "axis_config": {
                 "color": GREY,
             },
```

Comment:
I wonder whether these two entries should be merged, instead of just deleting one of them.

Reply:
I really don't know what they did, and flake reported the first one as pointless since it got overwritten, and I agreed.